mne/preprocessing/ica.py

```diff
@@ -548,8 +548,7 @@ def _transform_epochs(self, epochs, concatenate):
         if not hasattr(self, 'mixing_matrix_'):
             raise RuntimeError('No fit available. Please fit ICA')
 
-        picks = pick_types(epochs.info, include=self.ch_names, exclude=[],
-                           ref_meg=False)
+        picks = [epochs.ch_names.index(k) for k in self.ch_names]
```

Comment:
it's currently 
`picks = pick_types(epochs.info, include=self.ch_names, exclude=[], ref_meg=False)`

that would be a simple fix instead of the list comprehension. should we then replace the list comp in the `_transform_raw`?